### DISCUSSION TITLE
How to load to a specified resource group when loading data By Milvus Java SDK？

### DISCUSSION THREAD

**Discussion by zhaizhong** [milvus-io/milvus#28013] [Q&A and General discussion] [answered]
**[enviroment]**
  Milvus-v2.2.11
  Milvus Java SDK 2.3.2
**[action]**
  I tried to load data into the specified resource group through the loadCollection method in Java MilvusServiceClient, but I did not find the corresponding parameters from the io.milvus.param.collection.LoadCollectionParam class.
  From the documentation [](https://milvus.io/docs/resource_group.md), I only see that the python SDK supports loading data into specified resource groups.
**[question]**
How to load to a specified resource group when loading data By Milvus Java SDK？Is there a supported restful API? Or I need to integrate the ability of milvus python SDK in my Java service?

**Comment by yhmo** [comment, marked as answer]
Loading resource groups is not supported by Java SDK, yet. I think the next minor version will support it.
